source/_includes/pup43_platforms_fedora.markdown

```diff
@@ -0,0 +1,7 @@
+We publish official packages and run automated testing on the following versions for Puppet 4.3:
+
+* Fedora 22
+* Fedora 21
+* Fedora 20
```

Comment:
Well, PC1 isn't a static thing. We'll add and remove platforms as we get support in and they go end of life. The release package should be removed, since we've officially stopped supporting the platform. The main reason it hasn't is that we *eventually* want to create an archive server to put all the old builds that were publicly available. But, we haven't done that yet :(